content/en/docs/ops/troubleshooting/istioctl-analyze/index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Analyze your cluster and local configuration with istioctl analyze
+description: Shows you how to use istioctl analyze to identify potential issues with your configuration.
+weight: 90
+keywords: [istioctl, debugging, kubernetes]
+---
+
+{{< boilerplate experimental-feature-warning >}}
+
+## Getting started in under a minute
+
+`Istioctl analyze` is a powerful Istio diagnostic command that you can get started with in no time, and on any cluster.
+
+First, download the latest `istioctl` into the current folder using one bash command:
+
+Mac:
+
+{{< text bash >}}
+$ curl https://storage.googleapis.com/istio-build/dev/latest | xargs -I {} curl https://storage.googleapis.com/istio-build/dev/{}/istioctl-{}-osx.tar.gz | tar xvz
+{{< /text >}}
+
+Linux:
+
+{{< text bash >}}
+$ curl https://storage.googleapis.com/istio-build/dev/latest | xargs -I {} curl https://storage.googleapis.com/istio-build/dev/{}/istioctl-{}-linux.tar.gz | tar xvz
+{{< /text >}}
+
+Then, run it against your current Kubernetes cluster:
+
+{{< text bash >}}
+$ ./istioctl x analyze -k
+{{< /text >}}
+
+And that’s it! It’ll give you any recommendations that apply.
+
+For example, if you forgot to enable Istio injection (very common issue), you would get the following warning:
+
+{{< text plain >}}
+Warn [IST0102](Namespace/default) The namespace is not enabled for Istio injection. Run 'kubectl label namespace default istio-injection=enabled' to enable it, or 'kubectl label namespace default istio-injection=disabled' to explicitly mark it as not needing injection
+{{< /text >}}
+
+Notes: the ‘x’ in the command is because it’s currently ‘eXperimental’. It will eventually just be `istioctl analyze`.
+
+## Analyzing live clusters, local files, or both
+
+The scenario in the ‘getting started’ section is doing analysis on live cluster. But the tool also supports performing analysis of a set of local yaml configuration files, or on a combination of local files and a live cluster.
+
+{{< text bash >}}
```

Comment:
Separate this into distinct `<text>` blocks. People should be able to click the Copy button and paste into their terminal, which wouldn't work with this combo approach.